### DISCUSSION TITLE
Non relativistic laser intensity

### DISCUSSION THREAD

**Discussion by dpkhatua** [epochpic/epoch#584] [General]
Hi all,
As we can see from the EPOCH manual, it is a fully relatistic PIC code. Can anyone suggest that if I use laser intensity way below the relativistic region, how accurate will the results be? Should we work in the relativistic region while using EPOCH?

Thanks.

**Comment by Status-Mirror** [comment]
Hi @Khatua03,

The relativistic equations of motion are still valid for non-relativistic particles, so I think it should be fine.

If you're dealing with a low temperature plasma, you may find you need a greater resolution to prevent self heating. A good test to check your parameters is given in our [self heating demo](https://epochpic.github.io/quickstart/basic_examples/particle_load.html#self-heating).

Hope this helps,
Stuart

**Reply by dpkhatua** [reply]
Hi Stuart,
Thank you so much for the reply and suggestion. Yes, you are right about the non-relativistic simulation. I read some papers where they used a 3D epoch for a 1e14 Wcm^-2 power laser.

Thank you.
With best regards,
Durga Prasad Khatua
